src/fiber/channel.rs

```diff
@@ -1264,9 +1206,42 @@ where
         state: &mut ChannelActorState,
         command: ShutdownCommand,
     ) -> ProcessingChannelResult {
-        // The force shutdown command has been handled speically in the `NetworkActorState#send_command_to_channel` function.
-        // We only need to handle the normal shutdown command here.
         debug!("Handling shutdown command: {:?}", &command);
+        if command.force {
+            match state.state {
+                ChannelState::ChannelReady() => {
```

Comment:
I'm not sure why we add force shutdown logic here, since it's moved into network actor in this PR: https://github.com/nervosnetwork/fiber/pull/261 . Now there are two copies of code since the related code in network actor is not deleted.

Reply:
When both nodes are online, if one of them force close the channel, current code only updates the actor state in the store, not the state in memory, which will cause the other node to continue to send it messages such as addtlc.